src/app/grid/grid-sample-selection-template-excel/grid-sample-selection-template-excel.component.scss

```diff
@@ -0,0 +1,56 @@
+@import "~igniteui-angular/lib/core/styles/themes/index";
+@include igx-core();
+@include igx-theme($default-palette);
+
+$default-arrow-color: igx-color($default-palette, 'grays', 400);
+$default-arrow-hover-color: igx-color($default-palette, 'grays', 800);
+$default-arrow-all-color: igx-color($default-palette, 'secondary', 100);
+$border-width: 12px;
+$border-style: solid;
+
+@mixin triangle($color) {
+    width: 0;
+    height: 0;
+    border-left: $border-width $border-style transparent;
+    border-top: $border-width $border-style transparent;
+    margin: 2;
+    border-right: $border-width $border-style $color;
+    border-bottom: $border-width $border-style $color;
+}
+
+@mixin selector($justify, $align) {
+    display: flex;
+    justify-content: $justify;
+    align-items: $align;
+    min-width: 50px;
+    cursor: pointer;
+}
+
+.header-selector {
+    @include selector(flex-end, flex-end);
+    height: 100%;
+    padding: 8px;
+
+    &:hover .excel-style-arrow {
+        @include triangle($default-arrow-hover-color);
+    }
+}
+
+.row-selector {
+    @include selector(center, center);
+}
+
+::ng-deep {
+    .igx-grid__cbx-selection {
```

Comment:
This tells me there's something wrong with the way the template is setup, this shouldn't be required from the users.

Reply:
we should remove this once it is fixed in the component default template